packages/demo/examples/08-network/ExpandableNetwork.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint class-methods-use-this: 0, react/no-unused-prop-types: 0 */
+// import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Network,
+} from '@data-ui/network';
+
+import { getAddedGraph } from './data';
+
+
+class ExpandableNetwork extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = this.getStateFromProps(props);
```

Comment:
can you just set this to `this.state = { graph: props.graph };` since it's so simple?